modules/installation-gcp-config-yaml.adoc

```diff
@@ -55,6 +55,9 @@ controlPlane: <2> <3>
             keyRing: test-machine-keys
             location: global
             projectID: project-id
+      tags: <6>
```

Comment:
I think these callout changes are good, but as whenever you have a large volume of callout changes + conditional statements, leave this alone for a while, grab some tea, and then triple check. :dizzy:

Reply:
Thanks Max, I took an other look a bit later and I think they're good to go.